tests/src/ViewModel/ButtonCollectionTest.php

```diff
@@ -24,19 +24,37 @@ public function it_has_data()
             'compact' => true,
         ];
 
-        $blockLink = new ButtonCollection([Button::link($data['buttons'][0]['text'], $data['buttons'][0]['path'])], $data['centered'], $data['compact']);
+        $blockLink = new ButtonCollection([Button::link($data['buttons'][0]['text'], $data['buttons'][0]['path'])], false, $data['centered'], $data['compact']);
 
         $this->assertSameWithoutOrder($data['buttons'], $blockLink['buttons']);
         $this->assertSame($data['centered'], $blockLink['centered']);
         $this->assertSame($data['compact'], $blockLink['compact']);
         $this->assertSame($data, $blockLink->toArray());
+
+        $data = [
+            'buttons' => [
+                [
+                    'text' => 'text',
+                    'classes' => 'button--default',
+                    'path' => '#'
+                ],
+            ],
+            'inline' => true,
+        ];
+
+        $blockAction = new ButtonCollection([Button::action($data['buttons'][0]['text'])], $data['inline']);
+
+        $this->assertSameWithoutOrder($data['buttons'], $blockAction['buttons']);
+        $this->assertSame($data['inline'], $blockAction['inline']);
+        $this->assertSameWithoutOrder($data, $blockAction->toArray());
     }
 
     public function viewModelProvider() : array
     {
         return [
             'minimum' => [new ButtonCollection([Button::link('text', 'path')])],
-            'complete' => [new ButtonCollection([Button::link('text', 'path')], true, true)],
+            'inline' => [new ButtonCollection([Button::action('text')], true)],
+            'centered and compact' => [new ButtonCollection([Button::link('text', 'path')], false, true, true)],
```

Comment:
Reintroduce complete with all bool's set to true.

Reply:
'complete' is now added with all values set to true, together with 'minimum', 'inline' and 'centered and compact'.